microsoftgraph/microsoft-graph-explorer-v2
# Fetches permission on method change

## Overview

Brief description of what this PR does, and why it is needed.
- Adds support for fetching permissions on method change.

### Demo

Optional. Screenshots, `curl` examples, etc.
![image](https://user-images.githubusercontent.com/40403681/67950474-b0e98300-fbfa-11e9-830e-d7d5466163a8.png)


### Notes

Closes https://github.com/microsoftgraph/microsoft-graph-explorer-v2/issues/266

## Testing Instructions

* Log on to GE -tst.
* Click on `my mail` sample query.
* Click on the Permissions tab.
* Select `POST` from the http method drop-down.
* Notice that the scope reads `Mail.ReadWrite`.
